domains/irr123.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "irr123",
+    "email": "ivan@bogomolov.work"
+  },
+  "records": {
+    "CNAME": "https://irr123.pages.dev"
```

CNAME doesnt allow trailing slashes (/)

```suggestion
    "CNAME": "irr123.pages.dev"
```